Sources/Clappr/Classes/Protocol/Plugin.swift

```diff
@@ -1,8 +1,15 @@
-public protocol Plugin: EventProtocol {
+public protocol Plugin: EventProtocol, Nameable {
     static var type: PluginType { get }
-    static var name: String { get }
-    var pluginName: String { get }
-    init()
     init(context: UIObject)
     func destroy()
 }
+
+public protocol Nameable {
```

Comment:
This should be named `NamedType` to be consistent with Android implementation.

Reply:
The presence of the word **Type** in the name of a protocol/interface looks strange since it already represents a type. It looks a little redundant. Can we keep the name `Nameable` and change in Android in the future?

Reply:
No, we should keep the consistency and reopen the discussion in the future.